flake.nix

```diff
@@ -173,6 +173,19 @@
             pname = name;
             cargoExtraArgs = "--bin ${name}";
           });
+
+          container = pkgs.dockerTools.buildLayeredImage {
+            name = name;
+            contents = [ package ];
+            config = {
+              Cmd = [
+                "${package}/bin/${name}"
+              ];
+              ExposedPorts = {
+                "${builtins.toString port}/tcp" = { };
```

Comment:
is this exposing port 8000? This `minimint` daemon currently takes 2 command line arguments: path to a config, and path to an embedded database. The config will say which port to bind to. We can change this behavior, too.

Reply:
Also, there will be 2 ports: one for communicating with other federation servers, and another for communicating with clients